### DISCUSSION TITLE
Ftp upload problem with MStar

### DISCUSSION THREAD

**Discussion by JozefLu** [roleoroleo/yi-hack-MStar#417] [General]
Hi,
My ftp upload after motion detection is not working regularilly.
Most frequently it is not uploaded to ftp and is still on SD card.
I have other Yi-hack cameras versions (Allwinner and Allwinner-v2) and they work fluently - everything is uploaded to ftp and then immediately deleted localy in SD card.
----
Jozef

**Comment by roleoroleo** [comment]
The three scripts are identical.
Check the network connection, maybe is wifi weaky?

**Reply by JozefLu** [reply]
wifi strength is 95%, but I will change to another access point to check

**Reply by JozefLu** [reply]
wifi changed, but problem still exist

**Comment by roleoroleo** [comment]
Try to run the script manually and post the output.

**Comment by blfriedman** [comment]
I have seen changes in 0.4.7 for ftp behavior - it now seems to send only at the beginning of the hour to send files for the prior hour.  Do you have many cameras?  When they all send at the same moment, it can overload the wifi segment causing some of them to fail.  I have another discussion about this change opened today.
